src/components/calculatorbutton.tsx

```diff
@@ -0,0 +1,18 @@
+import { ReactNode } from "react";
+
+interface Props {
+  children: ReactNode;
+  onClick: (value: number | string) => void;
+//   className: string;
+}
+
+export default function CalcButton({children, onClick}: Props) {
+  return (
+    <div
+      onClick={() => onClick(children)}
+      className="text-center bg-slate-100 active:shadow-sm active:text-sm px-3 py-3 rounded-md flex items-center justify-center shadow-md"
+    >
+      {children}
+    </div>
```

Comment:
For semantic purposes, this should be a `button` element not a `div`

Reply:
https://www.freecodecamp.org/news/semantic-html5-elements/

Reply:
I am on it.

Reply:
@ArinzeJeffrey-droid, I just replaced the div tags with button.